examples/protein-ligand/cli/protein-ligand.yaml

```diff
@@ -67,9 +67,13 @@ atom_selection: not water
 phases:
     - complex
     - solvent
-    - vacuum
+#    - vacuum
```

Comment:
Why comment this out?

Reply:
It was really for testing, it should be uncommented, thanks for catching it!